flume-ng-sinks/flume-ng-morphline-solr-sink/pom.xml

```diff
@@ -84,6 +84,11 @@ limitations under the License.
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.jdom</groupId>
+      <artifactId>jdom</artifactId>
```

Comment:
Do you happen to know if the license of `jdom` is ASF-compatible?

Reply:
This is what I've found:
1) https://github.com/hunterhacker/jdom/blob/3c4e2cfec3b8c7de45ec04cf5c2a01e3dc20c59c/maven/maven.pom#L31 says that its license is "Apache-like".
2) JDom is not explicitly excluded at https://www.apache.org/legal/resolved.html
3) There are many projects at https://github.com/apache that use JDom: https://github.com/search?q=org%3Aapache+org.jdom&type=code

I think it is fine to use it!

Reply:
Assuming https://issues.apache.org/jira/browse/LEGAL-204 is correct, we should be good.